Jenkinsfile_CNP

```diff
@@ -10,11 +10,27 @@ def component = "api"
 
 def branchesToSync = ['demo']
 
+def secrets = [
+  'darts-${env}': [
+    secret('GovukNotifyTestApiKey', 'GovukNotifyApiKey')
```

Comment:
i think the envVar needs to be whatever is in application.yaml.
Not sure if it will work with the dot in it, but it might be:-
`secret('GovukNotifyTestApiKey', 'darts.GovNotifyApiKey')`

Looking at SSCS code, they seem to reference the spring property as normal:-
![image](https://user-images.githubusercontent.com/131666106/236146595-f0e53d9e-19fb-4186-a852-64f262f3bf65.png)

Here it is defined in the application.yaml:-
![image](https://user-images.githubusercontent.com/131666106/236146796-9f2de191-e368-4f64-ae37-7c04d5702264.png)

Not sure why they have 2 different properties for the prod and test keys, i think we could handle that inside the profiles.

Then they set the secret here:-
![image](https://user-images.githubusercontent.com/131666106/236146988-3d0c51b7-e96d-4907-9044-320581d295bd.png)

and then the secrets are stored in the vault
![image](https://user-images.githubusercontent.com/131666106/236148161-c171000e-6e4c-45ae-a860-310a3dfa78e8.png)

Reply:
Yeah I was following the same thing, think it should be like that now? I don't believe it will work with a dot so I've taken that out.

Reply:
yup, looks good.